editor/editor_file_system.cpp

```diff
@@ -1102,6 +1110,20 @@ void EditorFileSystem::_process_file_system(const ScannedDirectory *p_scan_dir,
 	}
 }
 
+void EditorFileSystem::_process_removed_files_first_scan() {
+	for (const KeyValue<String, EditorFileSystem::FileCache> &kv : file_cache) {
+		if (!processed_files_first_scan.has(kv.key)) {
+			if (ClassDB::is_parent_class(kv.value.type, SNAME("Script"))) {
+				// A script has been removed on disk since the last startup.
+				// The documentation needs to be updated.
+				update_script_mutex.lock();
+				update_script_paths_documentation.insert(kv.key);
```

Comment:
I would also appreciate a quick comment here explaining why the script isn't added to `update_script_paths`. Maybe something like:
`// No need to add to update_script_paths since that is exclusively to identify global class names, handled elsewhere.`

Reply:
Good idea, I modified a bit your suggested comment to be even more precise. Done!